lib/modules/dosomething/dosomething_global/dosomething_global.module

```diff
@@ -247,7 +247,7 @@ function dosomething_global_convert_country_to_language($country) {
 function dosomething_global_get_language($account, stdClass $node = NULL) {
   if ($node) {
     // If there is a published translation for the node in the user's language, use it.
-    if ($node->translations->data['$account->language']['status']) {
+    if ($node->translations->data{$account->language}['status']) {
```

Comment:
Can we leave it as is for now and circle back to it later for refactoring at some point so we can move forward with testing in Thor?